cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -685,9 +685,21 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 			// this field has "genconversion=false" comment to ignore it.
 			continue
 		}
+		t := m.Type
+		if t.Kind == types.Alias {
+			copied := *t.Underlying
```

Comment:
> The only reason would be we think we need the authorization endpoints to be
> in protobuf.  Otherwise we don't.  @deads2k the *AccessReview endpoints -
> were you planning on using those?

Yeah, they should support protobuf.  Its useful for a webhook authenticator from a node back to master.  That way the node can run authorization checks remotely and efficiently.  We current do this in openshift and it makes sense in kube too.